api/swagger.yaml

```diff
@@ -7131,6 +7131,10 @@ paths:
             example:
               ID: "ak7w3gjqoa3kuz8xcpnyy0pvl"
               Warning: "unable to pin image doesnotexist:latest to digest: image library/doesnotexist:latest not found"
+        403:
+          description: "network not for services"
```

Comment:
should we use the actual response here (minus the name), e.g. `"network is not eligible for services"`?